### ISSUE TITLE
CGO dependency (`sqlite3-binding.c`) causing certain builds to fail

### ISSUE THREAD

**Issue by cmrunton** [cloudflare/cfssl#650]
When building using the [Go build script](https://github.com/cloudflare/cfssl/blob/4a47280f81d5234ae621bfff451a174af664c793/script/build) I get the following errors:

```
14 errors occurred:
--> darwin/386 error: exit status 1
Stderr: cmd/cfssl/cfssl.go:48:2: C source files not allowed when not using cgo or SWIG: sqlite3-binding.c

--> linux/arm error: exit status 1
Stderr: cmd/cfssl/cfssl.go:48:2: C source files not allowed when not using cgo or SWIG: sqlite3-binding.c

--> windows/386 error: exit status 1
Stderr: cmd/cfssl/cfssl.go:48:2: C source files not allowed when not using cgo or SWIG: sqlite3-binding.c

--> windows/amd64 error: exit status 1
Stderr: cmd/cfssl/cfssl.go:48:2: C source files not allowed when not using cgo or SWIG: sqlite3-binding.c

--> darwin/arm error: exit status 1
Stderr: cmd/cfssl/cfssl.go:48:2: C source files not allowed when not using cgo or SWIG: sqlite3-binding.c

--> darwin/amd64 error: exit status 1
Stderr: cmd/cfssl/cfssl.go:48:2: C source files not allowed when not using cgo or SWIG: sqlite3-binding.c
```

As noted in Issue #545 I removed [line 48 from `cmd/cfssl/cfssl.go`](https://github.com/cloudflare/cfssl/blob/master/cmd/cfssl/cfssl.go#L48) and the packages build fine. I don't need SQLite storage, so that isn't an issue for me, but I'm sure others might have a use for it. Let me know if you need any info on my build environment.

**Comment by rfc1459** [comment]
This is a regression caused by #562, which while adding support for MySQL effectively reverted the removal of the go-sqlite3 import in `cmd/cfssl/cfssl.go` (#546).

**Comment by rfc1459** [comment]
Scrap my previous comment: cross-compiling was fixed by #546, but it was broken _again_ with #587.
The MySQL PR has nothing to do with it - that was me being stupid (`git blame` instead of a proper bisect)
